### PR TITLE
Feature/events

### PR DESCRIPTION
**Pull request by krazemon**

### CODE REVIEW

File: api/handlers/combo.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      winston.log('info', 'feedItems.get called by ComboRouter');
+      Transaction.findPublic({}).then(function(transactions) {
+        JoinEvent.find().exec().then(function(evts) {
+          var combined = transactions.concat(evts);
+          combined.sort(function(a,b) {
+            return b.timeStamp - a.timeStamp;
+          });
+          return res.status(200).send(combined);
+        }, function(err) {
+          winston.log('error', 'Error finding transactions: %s', err);
```
Comment (ritmatter) [inline api/handlers/combo.js]: err.toString() or winston won't be able to print the err. Won't crash, but won't output the error

File: api/routes/TransactionRoutes.js [hunk 2]
```diff
@@ -146,9 +146,7 @@ module.exports = function(router, isAuthenticated, acl) {
   router.route('/transactions')
     // Get all the transactions, obscuring private fields
     .get(isAuthenticated, function(req, res) {
```
Comment (ritmatter) [inline api/routes/TransactionRoutes.js]: :)

File: api/routes/UserRoutes.js [hunk 3]
```diff
@@ -168,6 +168,9 @@ module.exports = function(router, isAuthenticated, acl) {
                 return res.status(200).end();
               }
             });
+
+            var fullName = req.body.firstname + ' ' + req.body.lastname;
```
Comment (ritmatter) [inline api/routes/UserRoutes.js]: Extra line?

**Comment by ritmatter** [comment]
LGTM, just get those comments addressed

**Comment by krazemon** [comment]
WOOT

**Pull request merged**